daemon/inspect.go

```diff
@@ -1,83 +1,106 @@
 package daemon
 
 import (
-	"encoding/json"
 	"fmt"
+	"time"
 
-	"github.com/docker/docker/engine"
+	"github.com/docker/docker/daemon/network"
 	"github.com/docker/docker/runconfig"
 )
 
-func (daemon *Daemon) ContainerInspect(job *engine.Job) error {
-	if len(job.Args) != 1 {
-		return fmt.Errorf("usage: %s NAME", job.Name)
-	}
-	name := job.Args[0]
+type ContainerJSON struct {
```

Comment:
as we discussion via irc - we can if we just have the stuff that appears on the wire in the api/type struct.